contracts/src/Contract.sol

```diff
@@ -58,5 +62,7 @@ contract Contract {
 
 		// Finally, execute your logic here, for example issue a token, NFT, etc...
 		// Make sure to emit some kind of event afterwards!
+
+		emit verified(nullifierHash);
```

Comment:
where is this event declared? why is it lowercase?

Reply:
event declared on line 31, capitalization fixed

thanks for flagging!